packages/dm-core-plugins/src/explorer/components/context-menu/NodeRightClickMenu.tsx

```diff
@@ -1,6 +1,7 @@
 import { TNodeWrapperProps } from '@development-framework/dm-core'
 import { Menu } from '@equinor/eds-core-react'
 import React, { useState } from 'react'
+import { EDialog } from '../../types'
```

Comment:
Should we maybe change the imports to be absolute rather than relative? 🤔

Reply:
Or, I mean, not something _to be fixed_ by this PR 😅

Reply:
Mh. My editor just auto-generate this for me, so I haven't given it much thought. I suppose the readability would be better, though